Resources/Prototypes/Entities/Structures/Walls/walls.yml

```diff
@@ -366,6 +366,8 @@
   - type: Appearance
     visuals:
       - type: ReinforcedWallVisualizer
+  - type: StaticPrice
+    price: 41.5 # total material cost. If you change the recipe for the wall you should recalculate this.
```

Comment:
a way to auto calc stuff like this for the future would be good but uhhhhhhh thats hard

Reply:
That'd require somehow walking through the construction graph and figuring out the cheapest route.
Very hard.